cfgov/v1/models/learn_page.py

```diff
@@ -106,6 +107,47 @@ class DocumentDetailPage(AbstractFilterPage):
     template = 'document-detail/index.html'
 
 
+class StoryPage(AbstractFilterPage):
```

Comment:
Can this page be added as a new Python module (separate file) instead of adding it to `learn_page.py`, which is already rather crowded?

More generally, are there additional task around other content or logic for migration of Everyone Has a Story that might motivate treating it as a separate app from `v1`?